test/task-checker.js

```diff
@@ -24,8 +24,9 @@
 require('chai').should();
 
 var sass = require('../src/wsk-tasks/sass.js');
+var babel = require('../src/wsk-tasks/babel.js');
```

Comment:
It would be better use `const` for require? e.g. `const babel = require('../src/wsk-tasks/babel.js')`